### ISSUE TITLE
Test for dominant player

### ISSUE THREAD

**Issue by jleclanche** [jleclanche/fireplace#278]
Figure out what to do with those.

``` python
def test_power_overwhelming_shadow_madness():
       game = prepare_game()
       game.player1.discard_hand()
       game.player2.discard_hand()

       # start the game on the dominant player
       if not game.players[0].current_player:
               game.end_turn()

       wisp1 = game.current_player.opponent.summon(SPELLBENDERT)
       shadowmadness = game.current_player.give("EX1_334")
       shadowmadness.play(target=wisp1)
       game.current_player.give("EX1_316").play(target=wisp1)
       game.end_turn()

       assert wisp1.dead

       wisp2 = game.current_player.opponent.summon(WISP)
       shadowmadness = game.current_player.give("EX1_334")
       shadowmadness.play(target=wisp2)
       game.current_player.give("EX1_316").play(target=wisp2)
       game.end_turn()

       assert not wisp2.dead
       assert wisp2.controller == game.current_player
```

**Comment by beheh** [comment]
Do they fail currently? Do you want them in a separate file?

**Comment by jleclanche** [comment]
It's for reference. I'm not sure I care about it yet.

**Comment by shinoi2** [comment]
However, note that the Shadow Madness + Power Overwhelming interaction has been fixed in Patch 5.0; the minion always die

https://hearthstone.fandom.com/wiki/Advanced_rulebook#Dominant_Player_Bug
